src/garage/tf/models/gaussian_lstm_model2.py

```diff
@@ -0,0 +1,264 @@
+"""GaussianLSTMModel."""
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from garage.tf.models import Model
+from garage.tf.models.lstm import lstm
+from garage.tf.models.parameter import recurrent_parameter
+
+
+class GaussianLSTMModel2(Model):
+    """GaussianLSTMModel.
+
+    Args:
+        output_dim (int): Output dimension of the model.
+        hidden_dim (int): Hidden dimension for LSTM cell for mean.
+        name (str): Model name, also the variable scope.
+        hidden_nonlinearity (callable): Activation function for intermediate
+            dense layer(s). It should return a tf.Tensor. Set it to
+            None to maintain a linear activation.
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        recurrent_nonlinearity (callable): Activation function for recurrent
+            layers. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        recurrent_w_init (callable): Initializer function for the weight
+            of recurrent layer(s). The function should return a
+            tf.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_state_init (callable): Initializer function for the
+            initial hidden state. The functino should return a tf.Tensor.
+        hidden_state_init_trainable (bool): Bool for whether the initial
+            hidden state is trainable.
+        cell_state_init (callable): Initializer function for the
+            initial cell state. The functino should return a tf.Tensor.
+        cell_state_init_trainable (bool): Bool for whether the initial
+            cell state is trainable.
+        forget_bias (bool): If True, add 1 to the bias of the forget gate at
+            initialization. It's used to reduce the scale of forgetting at the
+            beginning of the training.
+        learn_std (bool): Is std trainable.
+        init_std (float): Initial value for std.
+        std_share_network (bool): Boolean for whether mean and std share
+            the same network.
+        layer_normalization (bool): Bool for using layer normalization or not.
+
+    """
+
+    # pylint: disable=assignment-from-no-return
```

Comment:
which line?

Reply:
np.log(init_std)...

Reply:
I think it's a bug from their side

Reply:
please include a link to github showing the bug which caused it